tools/build/tslint-rules/importWithinPackageRule.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
```

Comment:
Can you delete all of the `tools/build/tslint-rules` folder? Looks like you may have picked it up from a previous pull.

Reply:
Deleted the `tools/build/tslint-rules` folder and its sub folders